### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bastoscorp**
### Type of Change
_Please check the relevant boxes by putting an "x" in the appropriate box._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update

### Description
Capture html content with xpath expression

### Related Issue(s)
_Please list any related issues or reference them._

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done._

- [X] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) document.
- [X] My code follows the code style of this project.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.
- [X] I have updated the [README.md](../README.md) as necessary if there are changes.
- [X] I have tested the changes on my local machine before submitting the PR.

### CODE REVIEW

File: core/types/scenario.go [hunk 1]
```diff
@@ -339,7 +340,7 @@ func validateCaptureConf(conf EnvCaptureConf) error {
 		}
 	}
 
-	if conf.From == Body && conf.JsonPath == nil && conf.RegExp == nil && conf.Xpath == nil {
+	if conf.From == Body && conf.JsonPath == nil && conf.RegExp == nil && conf.Xpath == nil && conf.XpathHtml == nil {
 		return CaptureConfigError{
 			msg: fmt.Sprintf("%s, one of json_path, regexp, xpath key must be specified when extracting from body", conf.Name),
```
Comment (kenanfarukcakir) [inline core/types/scenario.go:345]: Could you add xpath_html to error string?

File: core/scenario/scripting/extraction/html.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		return nil, err
+	}
+
+	// returns the first matched element
+	foundNode, err := htmlquery.Query(rootNode, xPath)
+	if foundNode == nil || err != nil {
+		return nil, fmt.Errorf("no match for the xPath_html: %s", xPath)
+	}
+
+	//return foundNode.InnerText(), nil
```
Comment (kenanfarukcakir) [inline core/scenario/scripting/extraction/html.go:26]: Please delete this comment.

Comment (bastoscorp) [reply core/scenario/scripting/extraction/html.go]: deleted

File: core/types/scenario.go (hunk 1 above)
Comment (bastoscorp) [reply core/types/scenario.go]: added

File: engine_docs/README.md [hunk 3]
```diff
@@ -680,7 +680,7 @@ If Ddosify can't receive the response for a request, that step is marked as Fail
 | `not`   | ( param `bool` ) | returns converse of given param |
 | `range`   | ( param `int`, low `int`,high `int` ) | returns param is in range of [low,high): low is included, high is not included. |
 | `json_path`   | ( json_path `string`) | extracts from response body using given json path |
-| `xml_path`   | ( xpath `string` ) | extracts from response body using given xml path |
```
Comment (kenanfarukcakir) [inline engine_docs/README.md:683]: This line should not change. "xml_path" is a function used in assertion.

**Pull request closed without merging**